gnomad/resources/resource_utils.py

```diff
@@ -318,16 +340,23 @@ class ExpressionResource(BaseResource):
 
     expected_file_extensions: List[str] = [".he"]
 
-    def he(self, force_import: bool = False) -> hl.expr.Expression:
+    def he(
+        self,
+        force_import: bool = False,
+        read_args: Optional[Dict[str, Any]] = None,
+    ) -> hl.expr.Expression:
         """
         Read and return the Hail Expression resource.
 
+        :param force_import: If ``True``, force the import of the resource even if it
+            already exists.
+        :param read_args: Any additional arguments to pass to hl.experimental.
```

Comment:
```suggestion
        :param read_args: Any additional arguments to pass to hl.experimental.read_expression.
```

Reply:
I like having adding `read_args` for consistency here, but I'm not sure we actually need it for HEs (the only other arg accepted right now is `_assert_type`) -- what do you think?

Reply:
I think it doesn't hurt to leave, then it's there if the HEs evolve to have more